tools/test-js-with-node

```diff
@@ -45,6 +45,7 @@ def make_set(files: list[str]) -> set[str]:
 EXEMPT_FILES = make_set(
     [
         "web/shared/src/poll_data.ts",
+        "web/shared/src/typeahead.ts",
```

Comment:
It's better to just have some tests call these new functions than add them to the exclude list.